apps/cowswap-frontend/src/modules/trade/pure/TradeConfirmation/index.tsx

```diff
@@ -143,6 +149,15 @@ export function TradeConfirmation(props: TradeConfirmationProps) {
           />
         </styledEl.AmountsPreviewContainer>
         {children}
+        {appData && (
+          <OrderHooksDetails appData={appData}>
+            {(children) => (
+              <ConfirmDetailsItem label="Hooks" tooltip="Hooks are interactions before/after order execution.">
+                {children}
+              </ConfirmDetailsItem>
+            )}
```

Comment:
Similar comment to the explorer changes: fn children instead of flat JSX

Reply:
But we need it here in order to wrap the content into `ConfirmDetailsItem`.
Also, Michel changed it a bit in [his PR](https://github.com/cowprotocol/cowswap/pull/4932)